content/en/blog/_posts/2021-04-21-Graceful-Node-Shutdown-Beta.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: blog
+title: 'Graceful Node Shutdown Goes Beta'
+date: 2021-04-21
+slug: graceful-node-shutdown-beta
+---
+
+**Authors:** David Porter (Google), Murnal Patel (Red Hat), and Tim Bannister (The Scale Factory)
+
+Graceful node shutdown, beta in 1.21, enables kubelet to gracefully evict pods during a machine shutdown.
```

Comment:
nit: terminology: should it be called node?

```suggestion
Graceful node shutdown, beta in 1.21, enables kubelet to gracefully evict pods during a node shutdown.
```

Reply:
Done.